modules/openthread/Kconfig.features

```diff
@@ -220,6 +223,27 @@ config OPENTHREAD_PLATFORM_NETIF
 config OPENTHREAD_PLATFORM_UDP
 	bool "Platform UDP support"
 
+config OPENTHREAD_POWER_SUPPLY_BATTERY
+	bool "OT_POWER_SUPPLY_BATTERY"
+
+config OPENTHREAD_POWER_SUPPLY_EXTERNAL
+	bool "OT_POWER_SUPPLY_EXTERNAL"
+
+config OPENTHREAD_POWER_SUPPLY_EXTERNAL_STABLE
+	bool "OT_POWER_SUPPLY_EXTERNAL_STABLE"
+
+config OPENTHREAD_POWER_SUPPLY_EXTERNAL_UNSTABLE
+	bool "OT_POWER_SUPPLY_EXTERNAL_UNSTABLE"
+
+config OPENTHREAD_POWER_SUPPLY
+	string
+	prompt "Power supply configuration"
+	default "BATTERY" if OPENTHREAD_POWER_SUPPLY_BATTERY
+	default "EXTERNAL" if OPENTHREAD_POWER_SUPPLY_EXTERNAL
+	default "EXTERNAL_STABLE" if OPENTHREAD_POWER_SUPPLY_EXTERNAL_STABLE
+	default "EXTERNAL_UNSTABLE" if OPENTHREAD_POWER_SUPPLY_EXTERNAL_UNSTABLE
+	default ""
```

Comment:
Make this a [`choice`](https://docs.zephyrproject.org/latest/build/kconfig/setting.html#configuring-choices)?

Reply:
Yes of course, I forgot to add.